frontend/src/pages/NewRunV2.tsx

```diff
@@ -334,6 +334,7 @@ function NewRunV2(props: NewRunV2Props) {
           }
           specParameters={specParameters}
           handleParameterChange={setRuntimeParameters}
+          setIsValidInput={setIsStartButtonEnabled}
```

Comment:
We need to consider all factors which determines startButton is enabled or not.

Take the reference below: We see that if errorMessage is not empty or templateString is empty, we will disable start button. So the `setIsValidInput` here will take a boolean from child component, if the boolean is false, we will disable startButton. But when the boolean is true, it doesn't mean we can enable startButton, we still need to check for the condition in the reference below too.

Reference: https://github.com/kubeflow/pipelines/blob/0358c2d58ef0bdd7a9f06d953d22c473e1788c11/frontend/src/pages/NewRunV2.tsx#L143-L166